### PR TITLE
Add NodeJS 20 support for Lambda

### PR DESCRIPTION
**Pull request by ataylorme**
## Motivation
I am upgrading my Lambdas to NodeJS 20, which is [now supported by AWS](https://aws.amazon.com/blogs/compute/node-js-20-x-runtime-now-available-in-aws-lambda/), as it is [the current LTS version](https://github.com/nodejs/Release) and want to test them with localstack

## Changes
Adds NodeJS 20 support for Lambda

## Testing
I was unable to test locally. I will spend more time on my local environment but any help testing and updating snapshots is appreciated

## TODO
- [ ] update test snapshots
- [ ] update documentation

### CODE REVIEW

**Comment by ataylorme** [comment]
I have read the CLA Document and I hereby sign the CLA

recheck

**Comment by ataylorme** [comment]
I have read the CLA Document and I hereby sign the CLA

File: localstack/aws/api/lambda_/__init__.py [hunk 1]
```diff
@@ -233,6 +233,7 @@ class Runtime(str):
     python3_11 = "python3.11"
     nodejs20_x = "nodejs20.x"
     provided_al2023 = "provided.al2023"
+    nodejs20_x = "nodejs20.x"
```
Comment (DavidTanner) [inline localstack/aws/api/lambda_/__init__.py:236]: Looks like this is a duplicate of line 234.

Comment (ataylorme) [reply localstack/aws/api/lambda_/__init__.py]: Good catch, updated in [`a5dec38`](https://github.com/localstack/localstack/pull/9659/commits/a5dec3830c0319414d7dacc912734825c4e0c947)

**Comment by ataylorme** [comment]
@joe4dev thank you for the detailed response. The more organized, single PR absolutely makes sense. I’m looking forward to it shipping in an upcoming release. Cheers!

**Comment by joe4dev** [comment]
@ataylorme You are welcome :)

The `latest` image is already out and the next tagged patch release will be published very soon (ETA: tomorrow).

Congratulations on your first LocalStack contribution 🥳

**Pull request closed without merging**